add Story ecosystem

jacob-tucker opened the pull request:
This PR adds the Story ecosystem into this repo.

Story is a purpose-built layer 1 blockchain designed specifically for intellectual property. You can register your IP on-chain and add usage terms to it in seconds, massively lowering the barrier to the currently complex & antiquated legal system for IP. For example, enforcing “you owe me 50% of your commercial revenue if you use my IP” without needing time, money, or lawyers.

Story Twitter: https://x.com/StoryProtocol

yizhao-ec commented:
Hey thanks for the PR! FYI - I made 1 small update to fix a GitHub org link, feel free to add the other repos in the org if you would like. 

Regarding the contents:

1. Story Protocol already exist here: https://crypto-ecosystems.xyz/#story-protocol - would prefer to delete the `ecoadd` in this PR to avoid duplicated ecosystems
2. the follow repos seem to be no longer public, and would request these to be removed since we can't validate Story protocol usage:
    * https://github.com/roschler/ippy-search-buildathon
    * https://github.com/csy100/youth-pai
    * https://github.com/Prnasin/health-data-guardian-verse
    * https://github.com/tapestryipco/tapestry-launchbot
    * https://github.com/mxber2022/LiveBlur.Enode
3. could you explain how these 2 repos interact with Story? 
    * https://github.com/Deepersensor/deepcoin
    * https://github.com/Ayushjain2205/omnipay

jacob-tucker commented:
Hi @yizhao-ec , thanks so much for reviewing this PR.

1. I have removed the `ecoadd` and instead modified this PR to add all the repos to the existing "Story Protocol" ecosystem
2. I have removed these
3. I have removed these as well

I couldn't find Story show up on the [Developer Report](https://www.developerreport.com/) which is why I was trying to add a new "Story" ecosystem. How can we get "Story Protocol" to show up on this developer report?

yizhao-ec commented:
We only show the top 100 ecosystems by full time devs (defined by "Developers who committed code 10+ days out of a month.") on the main site and the list is updated monthly. In the most recent update, Story Protocol was not high enough yet. 

Thanks for the updates! Will accept and merge.
